doc/examples/motion_planning_api/src/motion_planning_api_tutorial.cpp

```diff
@@ -185,16 +187,28 @@ int main(int argc, char** argv)
   req.group_name = PLANNING_GROUP;
   req.goal_constraints.push_back(pose_goal);
 
+  // Define workspace bounds
+  req.workspace_parameters.min_corner.x = req.workspace_parameters.min_corner.y =
+      req.workspace_parameters.min_corner.z = -5.0;
+  req.workspace_parameters.max_corner.x = req.workspace_parameters.max_corner.y =
+      req.workspace_parameters.max_corner.z = 5.0;
```

Comment:
Why do we need to do this when we didn't before? Does the request adapter have a different default behavior since before the refactor?

Reply:
Ah, then definitely keep it.